src/pages/webhooks/hooks.md

```diff
@@ -0,0 +1,183 @@
+
+```json
+{
+    "data": {
+        "product": {
+            "name": "string",
+            "sku": "string",
+            "qty": "float"
+        }
+    }
+}
+```
+
+The webhook contains a top-level `data` object, and a second-level `product` object with several fields. However, your remote application expects a payload with this structure:
+
+```json
+{
+    "product": {
+        "name": "string",
+        "sku": "string",
+        "quantity": "float"
+    }
+}
+```
+
+To transmit this object to the remote application, you'll need to remove the `data` object from the payload and rename `qty` to `quantity`. The `source` configuration attribute specifies the full path of a Commerce webhook field, while the `name` attribute defines the full path of the field to transmit. If the two values are identical, then you can omit the `source` attribute.
+
+The following example configures the webhook described above.
+
+```xml
+<hook name="validate_stock" url="https://example.com/product-validate-stock" timeout="2000" softTimeout="200" required="true" fallbackErrorMessage="Can't add the product to the cart right now">
+    <fields>
+        <field name='product.name' source='data.product.name' />
+        <field name='product.sku' source='data.product.sku' />
+        <field name='product.quantity' source='data.product.qty' />
+    </fields>
+</hook>
+```
+
+### Field converters
+
+You can implement a converter class to convert a field to a different data type. For example, Commerce stores order IDs as numeric values. If the hook endpoint expects order IDs to be text values, you must convert the numeric value to a string representation before sending the payload.
+
+All convertor classes must implement `Magento\AdobeCommerceWebhooks\Model\Filter\Converter\FieldConverterInterface`.
+
+```xml
+<fields>
+    <field name='order.id' source='data.order.id' />
+    <field name='order.status' source='data.order.status' converter="Path/To/The/Converter/Class" />
+</fields>
+```
+
+### Context fields
+
+You can add to the webhook payload values from the application context:
+
+```xml
+<fields>
+    <field name="customer.entity_id" source="data.customer.entity_id" />
+    <field name="customer.customer_email" source="context_customer_session.get_customer.get_email" />
+</fields>
+```
+
+In this example, the value of `customer.customer_email` will be set to `Magento\Customer\Model\Session::getCustomer()::getEmail()`. If the value does not exist or cannot be processed, the appropriate message will be logged, and the hook execution will not be interrupted.
+
+You can also specify the string arguments to use when accessing the application context. Provide these arguments within curly braces and delimit them with colons, if there are multiple arguments. The following example sets `config_value` to the value of `Magento\Framework\App\Config\ScopeConfigInterface::getValue('value/path', 'default')`.
```

Comment:
Does `if there are multiple arguments` apply to both the colons and the curly braces? Might want to clarify.